packages/financial-templates-lib/src/price-feed/PriceFeedMockScaled.js

```diff
@@ -2,74 +2,89 @@ const { toBN } = require("web3").utils;
 const { PriceFeedInterface } = require("./PriceFeedInterface");
 const { parseFixed } = require("@ethersproject/bignumber");
 
-// An implementation of PriceFeedInterface that medianizes other price feeds.
+// Adds a final precision conversion step to the PriceFeedMock before returning prices.
 class PriceFeedMockScaled extends PriceFeedInterface {
-  // Constructs the MedianizerPriceFeed.
-  // priceFeeds a list of priceFeeds to medianize. All elements must be of type PriceFeedInterface. Must be an array of
-  // at least one element.
-  constructor(currentPrice, historicalPrice, lastUpdateTime, invertPrice, decimals = 18) {
+  constructor(currentPrice, historicalPrice, lastUpdateTime, priceFeedDecimals = 18, lookback = 3600) {
```

Comment:
In this pricefeedMock, nothing, because its intended to use only for tests. But every `PriceFeed` that implements `PriceFeedInterface` has to implement `getLookback()` so this sets a value for that function. 

Generally, `lookback` sets the valid window into the past that a pricefeed can search for a historical price.